docs/knowledgebase/runtime/benchmarking.md

```diff
@@ -0,0 +1,547 @@
+---
+title: Runtime Benchmarking
+---
+
+## What Is Runtime Benchmarking?
+
+In Substrate, each block must be produced within a certain time limit to keep the blockchain healthy
+and functional. This is the target blocktime. So in each block production phase, Substrate can only
+take so many extrinsic calls and compute for their results. We use weight to represent how much
+computations are needed for a runtime extrinsic.
+
+Some extrinsics has more computation and has a larger weight. If this extrinsic is included, then
+there is less capacity left to include other extrinsics in the block. This is further explained in
+the [Transaction Weight chapter](../learn-substrate/weight).
+
+In Substrate, **10^12 Weight = 1 Second**, and i.e 1,000 weight = 1 nanosecond, measured on a
+referenced hardware.
```

Comment:
need reference to what is the _reference hardware_.

Reply:
added to the footnote.